### ISSUE TITLE
Optimization offline errors estimation terms computation

### ISSUE THREAD

**Issue by StephaneVeys** [feelpp/feelpp#323]
In order to provide an error estimation very quickly during the online step the framework CRB compute problems during the offline step and stock results in data structures.

Among them there is M_Gamma_pr. Let us see how to fill it. We suppose first that we deal with problem that are not using EIM.
Suppose that we want to solve `a(u,v;mu) = f(v;mu) \forall v \in Xh`.
Let (w,v)_X = a(w,v)

Some notations :
Let `Xsi_i , i = 1 ... N` the RB basis functions.
Let Qa be the number of terms linked to the bilinear form a(.,.) in the affine decomposition.
Via the parameter affine decomposition we have `a(u,v;mu) = \sum_{q=1}^{Qa} \theta_q(mu) a_q(u,v)`
M_Gamma_pr is a matrix of size Qa x Qa,
M_Gamma_pr[q1][q2] contains a matrix of size N x N.
For given index q1 and q2 the element (i,j) of the matrix is denoted (i,j)q1q2

The element (i,j)q1q2 is defined by `(i,j)q1q2 = (z1,z2)_X` where `z1` and `z2` are solution of

```
(z1 , v )_X = -a_q1( Xsi_i , v )   and (z2 , v )_X = -a_q2( Xsi_j , v )
```

And now look the element `(j,i)q2q1 = (z2,z1)_X` where `z1` and `z2` are defined above.
As `( z1 , z2 )_X = ( z2 , z1 )_X` we have 

```
M_Gamma_pr[q1][q2](i,j)  = M_Gamma_pr[q2][q1](j,i) 
```

In other words if we consider the super matrix M_Gamma_pr, each element (q1,q2) of this super matrix contains a matrix whose elements are (i,j)q1q2 described above.
We need to compute the lower part of M_Gamma_pr including the diagonal. But elements located upper the diagonal are obtained by applying a transpose
